### ISSUE TITLE
inference memory with torch.set_grad_enabled(True)

### ISSUE THREAD

**Issue by pwwwyyy** [Vchitect/Latte#99]
Thank you for your great work! For some reasons ,I need to infer with torch.set_grad_enabled(True),but 80G cuda memory is not enough. I want to know whether this is normal and the configuration you use when training transformer of Latte-1.

**Comment by maxin-cn** [comment]
You can enable [gradient checkpointing](https://github.com/Vchitect/Latte/blob/504ef3ee8bdeac94da01cd96b6df2a4211800b03/models/latte_t2v.py#L673)  to save the GPU memory.
